src/coreclr/scripts/superpmi_diffs.py

```diff
@@ -117,9 +107,15 @@ def __init__(self, coreclr_args):
         self.spmi_location = os.path.join(self.script_dir, "artifacts", "spmi")
 
         self.log_directory = coreclr_args.log_directory
-        self.host_os = coreclr_args.host_os
-        self.target_os = coreclr_args.target_os
-        self.arch_name = coreclr_args.arch
+        self.host_os = "windows" if platform.system() == "Windows" else "linux"
```

Comment:
what about mac?

Reply:
superpmi-diffs only supports Windows host (asmdiffs targetting all OS/archs, tpdiffs with MSVC) and Linux host (tpdiffs with Clang)